overlay/usr/bin/entrypoint

```diff
@@ -2,58 +2,60 @@
 
 set -eo pipefail
 
+source /usr/local/lib/log.sh
+
 /usr/local/bin/gomplate -V -o /etc/php7/php.ini -f /etc/templates/php.ini.tmpl
 /usr/local/bin/gomplate -V -o /var/www/app/application/config/config.php -f /etc/templates/config.php.tmpl
 
 if [ -f application/config/security.php ]; then
-    printf '\nFile security.php already provisioned ...\n'
+    log_info 'File security.php already provisioned'
 else
-    printf '\nCreating security.php ...\n'
+    log_info 'Creating security.php'
     if [ -n "$LIME_ENCRYPTION_KEYPAIR" ]; then
         /usr/local/bin/gomplate -V -o /var/www/app/application/config/security.php -f /etc/templates/security.php.tmpl
     else
-        printf 'No encryption keys was provided. A security.php config will be created by the application ...\n'
-        printf 'THIS FILE NEEDS TO BE PERSISTENT\n'
+        log_warn 'No encryption keys was provided. A security.php config will be created by the application'
```

Comment:
Just a small thing, could we be consistent with the use of single/double quotes?